src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Rectangle extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int length;
+    private int width;
+    private Random random = new Random();
```

Comment:
The 'random' instance should be a local variable in the 'randomizeProperty' method rather than a class-level field since it's only used in that method.